services/web/src/utils/README.md

```diff
@@ -82,6 +82,65 @@ import { formatUsd } from 'utils/currency';
 formatUsd(99.999999); // $99.99
 ```
 
+Convert an option / rich enum to a Label element:
+
+```javascript
+import { formatUsd } from 'utils/currency';
+const statuses = {
+  pending: {
+    name: 'Pending',
+    icon: 'wait',
+  },
+  scheduled: {
+    name: 'Scheduled',
+    icon: 'checkmark',
+    color: 'olive',
+  },
+  cancelled: {
+    name: 'Cancelled',
+    icon: 'dont',
+    color: 'red',
+  },
+};
+
+formatOption(statuses, 'scheduled'); // <Label color="olive" icon="checkmark"...
+```
+
+## Forms
+
+Quickly create a bunch of options for Dropdown (and Label) from a hash:
+
+```javascript
+import { createOptions } from 'utils/forms';
```

Comment:
what if we just called it `getDropdownOptions` and it was smart enough to handle anything you throw at it - objects, arrays of objects, or arrays of strings?

Reply:
Done-ish